pkg/cmd/create/create_test.go

```diff
@@ -62,7 +62,7 @@ func AssertPreview(t *testing.T, customService string) {
 
 	serviceName := customService
 	if serviceName == "" {
-		serviceName = repo
```

Comment:
Earlier we would have not been able to find any url because the code was guessing serviceName would be similar to 
o.Repository .